pom.xml

```diff
@@ -106,43 +107,6 @@
                     </exclusion>
                 </exclusions>
             </dependency>
-            <!-- Temporary until 0.24.x is released -->
```

Comment:
Wondering if I had added this specifically for that `check.skip-dependency-versions` 🤔 

I think even the `two-digits-backward-compatible` strategy isn't good enough for our `SNAPSHOT` scheme: https://github.com/killbill/killbill-oss-parent/blob/17b5048cafb6fe6c80a58ce12c6163844dea214c/pom.xml#L1712

The real fix would be:

1. Switch to https://github.com/basepom/dependency-versions-check-maven-plugin from the old Ning plugin (see https://github.com/killbill/killbill-oss-parent/pull/445 and https://github.com/killbill/killbill-oss-parent/pull/431#issuecomment-1112488332 for why I couldn't do it)
2. Submit a PR upstream to @hgschmie to support our naming scheme (`ignore-SNAPSHOT-with-sha` strategy)

Maybe take a look at this once the build is stable again?

Reply:
Tracked [here](https://github.com/killbill/killbill/issues/1763). Meanwhile, I'll mark this as resolved.